packages/documentation/src/components/Card.module.css

```diff
@@ -23,21 +23,51 @@
   background-color: var(--theme-color-2);
   margin: 0 1.25rem 1.25rem 0;
   cursor: pointer;
+  text-decoration: none;
+  transition: transform 0.15s ease-in-out;
+
+  @media (max-width: 684px) {
+    width: 100%;
+    margin-right: 0;
+  }
+
+  @media (min-width: 664px) and (max-width: 684px) {
+    max-width: 640px;
+  }
+
+  @media (min-width: 997px) and (max-width: 1087px) {
+    /*margin-right: 0;*/
```

Comment:
Is the code still necessary? If not please remove.

Reply:
removed in latest commit